addon/components/modal-dialog.js

```diff
@@ -14,111 +22,60 @@ const computedJoin = function(prop) {
 };
 
 export default Ember.Component.extend({
-  // modal-dialog is itself tagless. positioned-container provides the container div
   tagName: '',
-
-  layout: template,
-  modalService: injectService('modal-dialog'),
-  destinationElementId: reads('modalService.destinationElementId'),
-
-  useEmberTether: computed('modalService.useEmberTether', 'alignment', 'renderInPlace', function() {
-    return this.get('modalService.useEmberTether') && (this.get('alignment') !== 'none') && !this.get('renderInPlace');
+  layoutStrategy: computed('structure', 'useEmberTether', function() {
+    let strategy = get(this, 'structure');
+    const useEmberTether = get(this, 'useEmberTether');
+    if (strategy === 'tethered' && !useEmberTether) {
```

Comment:
We could make this more intentional if apps had to explicitly opt into a non-simple default with an application option... like `MODAL_DIALOG_USE_TETHER`